packages/icons/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@jpmorganchase/mosaic-icons",
```

Comment:
New package for icons, as previously icons were exported from theme package but we want the theme package to be CSS only (otherwise, the CSS build thinks they are server components)